code/obj/item/bowling.dm

```diff
@@ -72,6 +85,18 @@
 							src.hitWeak(hitMob, user)
 		return
 
+	proc/pre_attack_damage_modifier(obj/item/parent_item, atom/A, mob/user)
+		var/mob/living/carbon/human/human_user = user
+		if(human_user.w_uniform && istype(human_user.w_uniform, /obj/item/clothing/under/gimmick/bowling))
```

Comment:
```suggestion
		if(istype(human_user.w_uniform, /obj/item/clothing/under/gimmick/bowling))
```
`istype` already checks for `null` because `null` isn't an instance of any type